### DISCUSSION TITLE
Using a schema to display a form with a type list shows no elements

### DISCUSSION THREAD

**Discussion by DennisDarko** [vueform/vueform#81] [Questions] [answered]
### Environment

**package.json (npm)** 
{
 "nuxt": "^3.8.2",
 "@vueform/nuxt": "^1.4.0",
"@nuxt/vite-builder": "3.5.3",
 }

### Reproduction

https://codesandbox.io/p/sandbox/falling-sunset-3cw9d2

### Describe the bug

When using the schema variant, no form elements are output and the data object also remains empty.

**Comment by adamberecz** [comment, marked as answer]
When using multiple elements in the list, the container object needs to be wrapped in an `object` property:

```js
mylist: {
      type: "list",
      label: "To-do (schema variant - not working)",
      object: {
        label: 'List item label',
        schema: {
          todo: { type: "text", placeholder: "To-do" },
          ready: { type: "checkbox", text: "Ready" },
        },
      }
    },
```

This way you can also add properties to the object.
